index.html

```diff
@@ -16,50 +16,61 @@
 <header>
   <nav>
     <a href="#" class="logo">John Doe</a>
+    <div id="nav-b-wrap">
+      <a href="#">Portfolio</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+      <a id="letter-icon" href="#">
+        <img  src="./Images/letter.svg">
+      </a>
+    </div>
     <div id="menu-button"><img id="humberger" src="./Images/Menu humberger icon.svg"></div>
   </nav>
 </header>
 
 <section id="first">
   <div id="fs-content">
-    <h1 id="greeting">Hey there. I'm Alex</h1>
+    <h1 id="greeting">Hey there. <br>I'm Alex</h1>
     <h3>I am a Software Developer</h3>
     <p>
-      I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+      I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
     </p>
-    <ul id="soc-med">
-      <li>
-        <a href="#">
-          <img src="./Images/Github.svg" alt="GitHub">
-        </a>
-      </li>
-      <li>
-        <a href="#">
-          <img src="./Images/LinkedIn.svg" alt="LinkedIn">
-        </a>
-      </li>
-      <li>
-        <a href="#">
-          <img src="./Images/Peace-sign.svg" alt="Peace-sign">
-        </a>
-      </li>
-      <li>
-        <a href="#">
-          <img src="./Images/Twitter.svg" alt="Twitter">
-        </a>
-      </li>
-      <li>
-        <a href="#">
-          <img src="./Images/M.svg" alt="M">
-        </a>
-      </li>
-    </ul>
+    <div id="soc-med-wrap">
+      <ul id="soc-med">
+        <li>
+          <a href="#">
+            <img src="./Images/github.svg" alt="GitHub">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./Images/linkedIn.svg" alt="LinkedIn">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./Images/peace-sign.svg" alt="Peace-sign">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./Images/twitter.svg" alt="Twitter">
+          </a>
```

Comment:
- [ ] Good job so far 👏 👏. your desktop design looks great. However, even if your images may be visible on your local machine, there are issues of case sensitivity that may make your images look broken when you deploy your site on some hosting platforms. 
Currently, this is how they appear 👇 
![image](https://user-images.githubusercontent.com/90222110/193040396-f0e6d7d5-1dce-443a-ab02-b6bc22cd8498.png)
This is because you have not referenced the correct image path; for the image path, we should be accurate as much as possible 😃 👍
Take a look at how the images are named here 👇 
![image](https://user-images.githubusercontent.com/90222110/193041282-096ece7a-9a59-42db-a879-a6282208f3b4.png)

Please correct it 🏃

Reply:
In today's morning I thought about problems with case sensitivity and renamed all images in my local directory to fully lowercase as well as changed every file path in my code to lowercase too.
Hm, looks like for remote repo my renaming doesn't really matter if after ten commits files here are with unchanged names.
So, what can I do to change them here too? 
Cuz on my machine everything now matches.